File: 백준/Bronze/28453. Previous Level/Previous Level.py
ran = int(input())
num = list(map(int, input().split()))

for n in num:
    if n == 300:
        print(1, end=" ")
    elif n >= 275:
        print(2, end=" ")
    elif n >= 250:
        print(3, end=" ")
    else:
        print(4, end=" ")

File: 백준/Bronze/28453. Previous Level/Previous Level.cs
int ran = int.Parse(Console.ReadLine());

string[] str = Console.ReadLine().Split();
int[] num = new int[ran];

for (int i = 0; i < ran; i++)
{
    num[i] = int.Parse(str[i]);

    if (num[i] == 300)
    {
        Console.Write("1 ");
    }
    else if (num[i] >= 275)
    {
        Console.Write("2 ");
    }
    else if (num[i] >= 250)
    {
        Console.Write("3 ");
    }
    else if (num[i] < 250)
    {
        Console.Write("4 ");
    }
}